cmd/api-router.go

```diff
@@ -34,67 +32,69 @@ func registerAPIRouter(mux *router.Router) {
 
 	// API Router
 	apiRouter := mux.NewRoute().PathPrefix("/").Subrouter()
+	var routers []*router.Router
+	if globalDomainName != "" {
+		routers = append(routers, apiRouter.Host("{bucket:.+}."+globalDomainName).Subrouter())
```

Comment:
Gateway router has router added as 

```
routers = append(routers, apiRouter.Host("{bucket:[A-Za-z0-9]+}."+globalDomainName).Subrouter())
```

While here it is `"{bucket:.+}"`, is there a reason for this difference

Reply:
good catch @nitisht gateway-router needs to be corrected